prowler/providers/azure/services/sqlserver/sqlserver_minimal_tls_version/sqlserver_minimal_tls_version.py

```diff
@@ -0,0 +1,22 @@
+from prowler.lib.check.models import Check, Check_Report_Azure
+from prowler.providers.azure.services.sqlserver.sqlserver_client import sqlserver_client
+
+
+class sqlserver_minimal_tls_version(Check):
+    def execute(self) -> Check_Report_Azure:
+        findings = []
+        for subscription, sql_servers in sqlserver_client.sql_servers.items():
+            for sql_server in sql_servers:
+                report = Check_Report_Azure(self.metadata())
+                report.subscription = subscription
+                report.resource_name = sql_server.name
+                report.resource_id = sql_server.id
+                report.status = "FAIL"
+                report.location = sql_server.location
+                report.status_extended = f"SQL Server {sql_server.name} from subscription {subscription} has no or an deprecated minimal TLS version set."
```

Comment:
```suggestion
                report.status_extended = f"SQL Server {sql_server.name} from subscription {subscription} is using TLS version {sql_server.minimal_tls_version} as minimal accepted which is not recommended. Please use one of the recommended versions: {<Here add the minimal recommended or the versions accepted>}"
```

Reply:
Yeah, I could do that. But then the value could be empty when nothing is set and the message would look like this:

```
SQL Server SQL Server Name from subscription 3bb43877-ccac-4ebd-8744-061f2808ad57 is using TLS version  as minimal accepted which is not recommended. Please use one of the recommended versions: 1.2, 1.3.
```

Reply:
What do you thing @puchy22. Should I implement it like that or use two different warning messages for lower TLS versions or no configured version?

Reply:
Hi, sorry for taking so long to reply and thanks for reminding me by tagging me back in the conversation. I think you should be able to follow my recommendation without problem since the `minimal_tls_version` variable should always be on the server, at least Azure reflects it in the [model documentation](https://learn.microsoft.com/en-us/python/api/azure-mgmt-sql/azure.mgmt.sql.models.server?view=azure-python) since it is a required parameter in its construction.

Reply:
Hi @puchy22, fine with me. Then this is already implemented, please have a look.